app/views/pages/_editor_guide_text.html.erb

```diff
@@ -9,33 +9,56 @@
     visibility: visible;
   }
 
+  .container .body pre {
+    margin-bottom: 1em;
+  }
 </style>
-<div class="container article" id="editor-help-guide">
-  <div class="title help-guide-title">
-    <h1>Editor Guide 🤓</h1>
-  </div>
-  <div class="body">
+
+<main class="container article" id="editor-help-guide">
+  <article id="main-content" class="body">
+    <h1 style="font-size: 57px; font-weight: 700; line-height: 73px;">Editor Guide 🤓</h1>
```

Comment:
I know, it's inline markup but the whole page sould be "crayonsified" and it would have been a much longer refactoring